brave-core/src/main/java/com/github/kristofa/brave/LoggingReporter.java

```diff
@@ -0,0 +1,38 @@
+package com.github.kristofa.brave;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import zipkin.reporter.Reporter;
+
+import static com.github.kristofa.brave.internal.Util.checkNotBlank;
+import static com.github.kristofa.brave.internal.Util.checkNotNull;
+
+/**
+ * Simple {@link SpanCollector} implementation which logs the span through jul at INFO level.
```

Comment:
Did you mean to still link to `SpanCollector` here instead of `zipkin.reporter.Reporter`?